### DISCUSSION TITLE
Configure Semantic-Release to only show the latest package update in release notes

### DISCUSSION THREAD

**Discussion by Pho3njx** [semantic-release/semantic-release#3324] [Configuration] [answered]
Hi everyone, 

I would like to ask for your support. I wasn't able to find any similar info in the docs. But maybe you can help me.

I'm using both **Renovate** (to update deps) and **Semantic-Release**. Apps and NPM libs are stored on GitLab and have a `.releaserc` config file. I use `conventionalcommits` as preset for the semantic release.

The following release notes are written to the CHANGELOG.md after a semantic release:
```
### Chores
    * deps: update dependency @types/node to v20.12.10      
    * deps: update dependency @types/node to v20.12.11      
    * deps: update dependency @types/node to v20.12.12      
    * deps: update dependency @types/node to v20.12.8       
    * deps: update dependency @types/node to v20.12.9       
    * deps: update typescript-eslint monorepo to v7.10.0    
    * deps: update typescript-eslint monorepo to v7.8.0     
    * deps: update typescript-eslint monorepo to v7.9.0
    * [...and so on...]
``` 

Every Renovate update is shown in the release notes. Of course, every Renovate update generates a semantic commit message in the main branch. And so, they all will be taken into account during a semantic release. But I think that's not needed in such detail. It would be enough only to write the latest update of a package in the release notes.

Is it possible to configure Semantic-Release that only the latest update of a package will be written in the release notes?
**_Example:_**
```
### Chores
    * deps: update dependency @types/node to v20.12.12      
    * deps: update typescript-eslint monorepo to v7.10.0    
    * [...and so on...]
```

Thank you so much for your help and best regards!

**Comment by travi** [comment]
the chore commit type is not included in the release notes by default because it is not a change that directly impacts your consumers. if you are concerned about chores adding too much noise to your release notes, why have you chosen to include that commit type in the release notes in the first place

**Comment by Pho3njx** [comment]
I agree in general. Chores mostly don't impact the consumers. Sure, I could think about it and remove chore commits from release notes and that would fix my problem. I also could write the CHANGELOG.md manually by myself. You could also ask why I use pinned dependencies in the package.json instead of using SemVer ranges (^ or ~). That would also fix it, because Renovate wouldn't make so much noise. But it all doesn't answer my question.

So, do I understand your reply correct, that Semantic Release is currently **not** able  (e.g. by config or by a plug-in) to "merge" such commits, so that only the latest update of a package is mentioned in the release notes?

**Reply by travi** [reply, marked as answer]
each commit is analyzed individually. there is no process available to perform additional logic for behaviors across multiple commits

**Reply by Pho3njx** [reply]
Hi @travi, 
ok, many thanks for your answer 👍
